django/contrib/auth/validators.py

```diff
@@ -9,8 +9,8 @@
 class ASCIIUsernameValidator(validators.RegexValidator):
     regex = r"^[\w.@+-]+\Z"
     message = _(
-        "Enter a valid username. This value may contain only ASCII letters, "
-        "numbers, and @/./+/-/_ characters."
+        "Enter a valid username. This value may contain only unaccented lowercase a-z "
+        "and uppercase A-Z letters, numbers, and @/./+/-/ characters."
```

Comment:
```suggestion
        "and uppercase A-Z letters, numbers, and @/./+/-/_ characters."
```
We lost the underscore, is that needed?

Reply:
Yes we need, great catch :dart: